packages/services/api/src/shared/helpers.ts

```diff
@@ -7,7 +7,7 @@ import type {
   OrganizationSelector,
   ProjectSelector,
   TargetSelector,
-} from '../__generated__/types';
+} from '../__generated__/types.next';
```

Comment:
I'm seeing a TS error if I use the old generated `types` file here. The reason being there's a type that overrides the Node's `Date` type being generated at the bottom of the file:

```ts
export type Date = Scalars["Date"];
```

Switching to the new types seems to fix this issue, so I think we can start using it